modules/EC2ImageBuilderGoldenAMI/README.md

```diff
@@ -224,3 +230,115 @@ The _EC2 Image Builder GoldenAMI_ example module, by default:
+
+```shell
+cfn generate && cfn submit --set-default --region us-east-1
+```
+
+
+### Using StackSets
+You'll use, in this example, the
+`example/module-deployment-stack.template` sample CloudFormation
+template that describes deployment-related resources for this module
+with code; you'll use StackSets to deploy the module across AWS
+regions you need.  Steps:
+
+- Use the CloudFormation CLI to create a ZIP archive for this module's
+  content:
+
+```shell
+cfn generate && cfn submit --dry-run
+```
+
+- The dry-run above should have created the
+  `awssamples-ec2imagebuilder-goldenami-module.zip` archive in the
+  same directory as this `README.md` file.  Upload the ZIP archive to
+  an S3 bucket you own.  Use a method of your choice, such as the S3
+  console or the AWS CLI; for more information, see [Uploading
+  objects](https://docs.aws.amazon.com/AmazonS3/latest/userguide/upload-objects.html).
+
+- Next, upload the `example/module-deployment-stack.template` template
+  file to your bucket as well.
+
+- Prepare your account for it to work with StackSets, by referring to
+  [Prerequisites for stack set
+  operations](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-prereqs.html).
+  For example, if you choose to use [Self-managed
+  permissions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-prereqs-self-managed.html#prereqs-self-managed-permissions),
+  refer to [Set up basic permissions for stack set
+  operations](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-prereqs-self-managed.html#stacksets-prereqs-accountsetup)
+  to create `AWSCloudFormationStackSetAdministrationRole` and
+  `AWSCloudFormationStackSetExecutionRole` roles with provided
+  templates.
+
+- Next,
+  [create](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-getting-started-create.html)
+  your StackSet.  As an example, the following process describes next
+  steps if you choose to use the AWS CloudFormation
+  [console](https://console.aws.amazon.com/cloudformation/) and
+  [self-managed
+  permissions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-getting-started-create.html).
+
+- From the navigation pane in the console, choose **StackSets**, and
+  then **Create StackSet**.
+
+- In the next page, choose to use **Self-service permissions** and,
+  where indicated, specify the names of the roles you created
+  previously (`AWSCloudFormationStackSetAdministrationRole`, and
+  `AWSCloudFormationStackSetExecutionRole` respectively).
+
+- In the **Prerequisite - Prepare template** section, specify the
+  Amazon S3 URL of the `module-deployment-stack.template` template you
+  uploaded to your bucket earlier.  Alternatively, choose to upload
+  the `module-deployment-stack.template` template file.
+
+- Specify a name for your StackSet.
+
+- For the `ModulePackage` parameter, specify the URL of the ZIP file
```

Comment:
Potentially we could modify `URL` to `Amazon S3 URL` (similar to what was done 2 steps before), to make it clear that an S3 URL would need to be specified?

Reply:
Thank you, @arthurboghossian - done!